### DISCUSSION TITLE
tutorials/programmability/json-rpc/ansible/

### DISCUSSION THREAD

**Discussion by giscus** [srl-labs/learn-srlinux#50] [comments]
# tutorials/programmability/json-rpc/ansible/

Open documentation for open Network OS

https://learn.srlinux.dev/tutorials/programmability/json-rpc/ansible/

**Comment by hellt** [comment]
As was pointed out (https://twitter.com/ganesh634/status/1600933038132629504) building hermetic ansible environments can be done with https://ansible-builder.readthedocs.io/en/latest/
